elcid/assets/js/elcid/services/flow.js

```diff
@@ -33,6 +33,12 @@ angular.module('opal.services').factory('elCIDFlow', function($routeParams){
                         'template'  : '/templates/modals/hospital_number.html/'
                     }
                 }
+                if($routeParams.slug && $routeParams.slug === 'virology'){
+                  return {
```

Comment:
Reckon we could add a test for this ?

We've added the condition but not tested it etc....